SearchEngines.yml

```diff
@@ -1355,8 +1355,6 @@ IxQuick:
     params:
       - query
     backlink: 'do/asearch?query={k}'
-    hiddenkeyword:
-      - '/do/asearch'
```

Comment:
that would actually mean that e.g. `ixquick.com/do/asearch` would not be detected anymore, as only `ixquick.com` and `ixquick.com/` would be allowed for hidden keywords

Reply:
This is intentional, as I can find nothing on startpage.com that uses `/do/asearch` anymore (the links to the website are directly in the `<a>` and I doubt they use it anymore if the referrer policy does the same result but without the disadvantages of a redirect.

Reply:
Sure. Then it might only make a difference when processing historical data